WDL.YAML-tmLanguage

```diff
@@ -9,13 +9,13 @@ patterns:
   match: \=
 
 - name: keyword.operator.comparison.wdl
-  match: <\=|>\=|\=\=|<|>|\!\=
+  match: <\=|>\=|\=\=|<{1}|>{1}|\!\=
```

Comment:
I'm curious whether these are needed - unless they enforce not having two in a row this seems identical to what was there before?

Reply:
I was being a bit overzealous! Will remove.